formats/dot/internal/errors/errors.go

```diff
@@ -2,7 +2,7 @@
 // Use of this source code is governed by a BSD-style
 // license that can be found in the LICENSE file.
 
-// generated by gocc; DO NOT EDIT.
+// Code generated by gocc; DO NOT EDIT.
```

Comment:
It seems that github is stricter than the decision for this form. This diff should not be visible - see for example https://github.com/gonum/blas/commit/0285886d824b705f344567a692d06cf96179ad67. It probably needs to be on the first line.